Sources/BuildServerProtocol/Messages/OnBuildLogMessageNotification.swift

```diff
@@ -14,23 +14,161 @@ public import LanguageServerProtocol
 
 /// The log message notification is sent from a server to a client to ask the client to log a particular message in its console.
 ///
-/// A `build/logMessage`` notification is similar to LSP's `window/logMessage``.
+/// A `build/logMessage`` notification is similar to LSP's `window/logMessage``, except for a few additions like id and originId.
 public struct OnBuildLogMessageNotification: NotificationType {
   public static let method: String = "build/logMessage"
 
   /// The message type.
   public var type: MessageType
 
+  /// The task id if any.
+  public var task: TaskId?
+
+  /// The request id that originated this notification.
+  /// The originId field helps clients know which request originated a notification in case several requests are handled by the
+  /// client at the same time. It will only be populated if the client defined it in the request that triggered this notification. */
```

Comment:
```suggestion
  /// client at the same time. It will only be populated if the client defined it in the request that triggered this notification.
```